corehq/apps/accounting/management/commands/add_operations_user.py

```diff
@@ -11,12 +11,19 @@
 class Command(BaseCommand):
     help = 'Grants the user(s) specified the DIMAGI_OPERATIONS_TEAM privilege'
 
-    option_list = (
-        make_option('--remove-user', action='store_true',  default=False,
-                    help='Remove the users specified from the DIMAGI_OPERATIONS_TEAM privilege'),
-    )
+    def add_arguments(self, parser):
+        parser.add_arguments(
+            'usernames',
+            nargs="*",
```

Comment:
Ah, this confused me for a minute - I was expecting `*args` to be a single argument with the possibility of more.